packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Changed
+
+- **BREAKING:** `getSelectedAddress` is replaced with `getSelectedAccount` in the `TransactionController` ([#4244](https://github.com/MetaMask/core/pull/4244))
```

Comment:
This makes it sound like this option was replaced, but that's not true; this option was removed, and now `AccountsController:getSelectedAccount` is used to grab the currently selected address. Perhaps we ought to replace this entry with:

``` suggestion
- **BREAKING:** The `TransactionController` messenger must now allow the `AccountsController:getSelectedAccount` action ([#4244](https://github.com/MetaMask/core/pull/4244))
```

And then add this to Removed:

``` markdown
- Remove `getSelectedAddress` option from `TransactionController`  ([#4244](https://github.com/MetaMask/core/pull/4244))
  - The AccountsController is used to get the currently selected address automatically.
```